readme.md

```diff
@@ -7,7 +7,8 @@ The following packages are available:
 - [Uno.Microsoft.Toolkit](https://www.nuget.org/packages/Uno.Microsoft.Toolkit )
 - [Uno.Microsoft.Toolkit.Parsers](https://www.nuget.org/packages/Uno.Microsoft.Toolkit.Parsers)
 - [Uno.Microsoft.Toolkit.Services](https://www.nuget.org/packages/Uno.Microsoft.Toolkit.Services)
-- [Uno.Microsoft.Toolkit.Notifications](https://www.nuget.org/packages/Uno.Microsoft.Toolkit.Notifications)
+- [Uno.Microsoft.Toolkit.UWP.Notifications](https://www.nuget.org/packages/Uno.Microsoft.Toolkit.Uwp.Notifications)
+- [Uno.Microsoft.Toolkit.UWP.Notifications.JavaScript](https://www.nuget.org/packages/Uno.Microsoft.Toolkit.Uwp.Notifications.JavaScript)
```

Comment:
I think this link was omitted because it's not important for Uno, WinJS is an unrelated technology. @jeromelaban might confirm.

Reply:
We might leave the Javascript-Nuget out - I don't know. It's just that the previous "Notifications"-link lead to a 404. I don't know which package(s) is/are relevant here.

Reply:
There's no problem updating those links. Those packages are rebuilt to match with the Uno prefix, but that's pretty much it.